### DISCUSSION TITLE
Create EDC installation guide

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-tractusx/tractusx-edc#270] [Ideas]
**WHAT**
Create a documentation on how to deploy a Catena-X EDC via helm charts from scratch (e.g. to azure). This documentation should be used as installation guide for companies willing to deploy EDC. Outcome should be aligned with Catena-X standards and components. Documentation deployed in Tractus-X.

**WHY**
Various documentations at different locations exists, which makes it difficult to deploy an EDC Connector for Catena-X. 

**SOLUTION PROPOSAL**
-	Step-by-step installation guide
-	Highlight basic requirements for installation (e.g. Java, Kubernetes, maven…)
-	Document prerequisites or external sources
-	Document required configurations, properties etc.

**FURTHER NOTES**
Continuous improvement of documentation planned. Starting with first minimal viable documentation which will be enhanced over time.

--

If such a guide already exist, please let us know and link it :) Otherwise we are happy to make our contribution and start with such a guide.
Also refers to https://github.com/eclipse-tractusx/tractusx-edc/issues/170

**Comment by stephanbcbauer** [comment]
@tmberthold not sure about the official discussion process :) But since you created an Issue #275 and also a PR was created ... can we close this discussion? We could also wait until the PR is merged and link the related files as a result here
